crates/core/thin_edge_json/src/parser.rs

```diff
@@ -99,8 +99,11 @@ where
                 }
                 "time" => {
                     let timestamp_str: &str = map.next_value()?;
-                    let timestamp = DateTime::parse_from_rfc3339(timestamp_str)
-                        .map_err(|err| de::Error::custom(invalid_timestamp(timestamp_str, err)))?;
+                    let timestamp = OffsetDateTime::parse(
```

Comment:
Wondering if this mapping logic, which is used at multiple places (including tests), can be moved into the common `clock` module for seamless re-use.

Reply:
I suppose it could, but the goal was to remove `chrono` and not refactor the module, I may propose this change in separate pr, though.